notes/force-move-implementation.md

```diff
@@ -27,14 +27,23 @@ struct ChannelStorage {
   address challengerAddress;
 }
 
-mapping(address => ChannelStatus) channelStatuses;
+mapping(address => ChannelStorage) channelStorages;
 ```
 
-**The key idea of this section is to store a commitment to this data, instead of storing the data itself.** The actual data can then be provided as required to each method, as part of the calldata.
+`turNumRecord` is the highest turn number that has been established on chain. Established here means being the highest turn number of a valid and visible n-chain (i.e. an ordered list of n states). Valid means each state in the list is a valid transition from its predecessor. Visible means submitted to the chain.
```

Comment:
It's not quite this. For example, a ForceMove could be submitted with one state signed by n people, which would also advance the `turnNumRecord`. I'm not quite sure how to describe this though. Maybe it's the highest turnNum which is "supported by n signatures" or something.

Reply:
Have had another go in a8eaac7 -- any better?